week-2-practise/level-1/2-validation.js

```diff
@@ -2,7 +2,7 @@
 // TIP: You should write an expression that returns a boolean value
 
 var num = 10;
-var isBigEnough; // ONLY EDIT THIS LINE
+var isBigEnough = true; // ONLY EDIT THIS LINE
```

Comment:
We wanted a boolean condition here, so that the log line will execute only if num is bigger than or equal to 10.